pkg/cmd/cli/install/install.go

```diff
@@ -179,11 +179,27 @@ func GetVeleroVersion(f client.Factory) (string, error) {
 // If currentVersion < minVersion, return -1
 // Assume input string format is vX.Y.Z
 func CompareVersion(currentVersion string, minVersion string) int {
+	if currentVersion == "" {
+		currentVersion = "v0.0.0"
+	}
+	if minVersion == "" {
+		minVersion = "v0.0.0"
+	}
 	curVerArr := strings.Split(currentVersion[1:], ".")
 	minVerArr := strings.Split(minVersion[1:], ".")
+
 	for index, _ := range curVerArr {
-		curVerDigit, _ := strconv.Atoi(curVerArr[index])
-		minVerDigit, _ := strconv.Atoi(minVerArr[index])
+		var curVerDigit, minVerDigit int
```

Comment:
Traversing curVerArr would be problematic in the case where curVerVersion is shorter than the minVersion. E.g,
currentVersion = v1 and minVersion = v1.2.3. This for loop would exit after one iteration and return 0, which means  two versions are equal.

We might want to traverse the array whichever is longer here.

Reply:
True.  I will change that